docs/self-host.docker

```diff
@@ -0,0 +1,16 @@
+FROM vladikoff/fxa-slim-image:latest
+
+RUN adduser --disabled-password --gecos '' fxa && adduser fxa sudo && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+COPY . /home/fxa/fxa-profile-server
+WORKDIR /home/fxa/fxa-profile-server
+RUN chown -R fxa .
+USER fxa
+
+RUN npm install --production \
+  && npm cache clear
+
+CMD npm start
+
+# Expose ports
+EXPOSE 1111
```

Comment:
@michielbdejong I tagged `1.0.0` here: https://hub.docker.com/r/vladikoff/fxa-slim-image/tags/

Can you updated this one? We can also update other repos with a small PR later.